outlines/publish-packages.md

```diff
@@ -0,0 +1,27 @@
+# Publishing a Package
+
+1. Pros/cons of publishing a package
+  1. Considering what's involved in publishing a package.
+  2. Do you think you'll be able to maintain it properly?
+  3. Is the code complete enough to be useful, or does it need work to be made generic?
+2. How to decide what's in a package / more than one
+  1. Is there any core functionality that could be reused elsewhere? (e.g. route matching)
+  2. Is there pure JS code that could be published to NPM?
+  3. Principle of a single symbol per package could help (see structure chapter).
+3. How to write great documentation.
+  1. README vs longer documentation/guide
```

Comment:
There's a crappy thing here which is that README table of contents doesn't work on Atmosphere. Perhaps we can fix this quickly? I feel like it would be like a 1 line fix to parse some links differently.